Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -759,6 +765,7 @@ will automatically configure your virtual network to route pod traffic correctly
      --set azure.clientSecret=AZURE_CLIENT_SECRET \\
      --set tunnel=disabled \\
      --set masquerade=false \\
+     --set devices=eth0 \\
```

Comment:
Do we know why `azure0` is detected instead of `eth0`? Is that something we should fix in the device detection logic?

Reply:
I haven't looked at it in detail. Can try to investigate further an would address in a follow-up PR in case there is something we could fix in the device detection logic.

Reply:
Agree with @pchaigno, seems strange that we need to start specifying this when we didn't have to before. Would be great if you could investigate and follow-up on this.